### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jamiebrynes7**
#### Description
* Added the `init.sh`/`init.ps1` steps for local development.
* Generally tidied up/updated other parts of the README.
* Added the latest release badge! 🎉 

I recommended checking out the [rendered version](https://github.com/spatialos/gdk-for-unity/tree/chore/update-readme/README.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,58 +1,91 @@
...
 
-**Github issues**<br/>
-Create an issue [in this repository](https://github.com/spatialos/UnityGDK/issues).
+* **Github issues**
+
+  Create an issue [in this repository](https://github.com/spatialos/UnityGDK/issues).
+
+## Contributions
+
+We are not currently accepting public contributions - see our [contributions](https://docs.improbable.io/unity/alpha/contributing) policy. However, we are accepting issues and we do want your feedback.
```
Comment (paulbalaji) [inline README.md]:
```suggestion	
We are not currently accepting public contributions - see our [contributions](https://docs.improbable.io/unity/alpha/contributing) policy. However, we are accepting issues and will gratefully appreciate your feedback.
```

File: README.md [hunk 2]
```diff
@@ -1,58 +1,91 @@
...
 
-* [Development roadmap](https://github.com/spatialos/gdk-for-unity/projects/1)
-* [Issue reporting](https://github.com/spatialos/UnityGDK/issues)
+## Developing on the GDK
 
-* Community: [Discord](https://discord.gg/SCZTCYm) - [Forums](https://forums.improbable.io/latest?tags=unity-gdk) - [Mailing list](http://go.pardot.com/l/169082/2018-06-25/27mhsb)
+We recommend that you start developing with the [**Blank Starter Project**](https://github.com/spatialos/gdk-for-unity-blank-project) or the [**First Person Shooter (FPS) Starter Project**](https://github.com/spatialos/gdk-for-unity-fps-starter-project/).
 
----
+If you'd like to develop on this repository, there is a little bit of additional setup:
```
Comment (paulbalaji) [inline README.md]: this new bit should be collapsible

Comment (jamiebrynes7) [reply README.md]: @jessicafalk probably will fight you over this

Comment (paulbalaji) [reply README.md]: resolved offline

File: README.md [hunk 3]
```diff
@@ -1,58 +1,72 @@
...
 
-This is the alpha release of the SpatialOS GDK for Unity. We invite projects to start using it but warn that all APIs are subject to change as we learn from feedback.
+## Having problems?
 
-## Contributions
+* [Documentation](https://docs.improbable.io/unity/alpha/)
+* [Troubleshooting](https://docs.improbable.io/unity/alpha/reference/troubleshooting)
+* [Known issues](https://github.com/spatialos/gdk-for-unity/projects/2)
+
+If you are unable to resolve your issue, please raise an issue [in this repository](https://github.com/spatialos/UnityGDK/issues) or find us on the [forums](https://forums.improbable.io/latest?tags=unity-gdk) or [Discord](https://discord.gg/SCZTCYm).
```
Comment (zeroZshadow) [inline README.md]: We should probably update the `in this repository` link

File: README.md [hunk 4]
```diff
@@ -1,58 +1,72 @@
...
 
 The SpatialOS Game Development Kit (GDK) for Unity allows you to quickly and easily build and host Unity multiplayer games. These games can use multiple server-side game engines across one seamless world to create new kinds of gameplay.
 
-The GDK is composed of three layers:
+If you're a Unity game developer and ready to try out the GDK, follow the [Get started guide](https://docs.improbable.io/unity/alpha/projects/fps/get-started/get-started).
 
-**The GDK Core:** a performant, data-oriented integration with our cloud platform SpatialOS, based on the familiar Unity-native workflows.
+> The SpatialOS GDK for Unity is in alpha. We are committed to rapid development of the GDK to improve stability, usability, and performance - for information on this, see our [development roadmap](https://github.com/spatialos/gdk-for-unity/projects/1) and contact us via our [forums](https://forums.improbable.io/latest?tags=unity-gdk), or on [Discord](https://discord.gg/SCZTCYm). Sign up to our [mailing list](http://go.pardot.com/l/169082/2018-06-25/27mhsb) to get updates about the GDK for Unity.
```
[the remark this answers is not in the document]
Comment (jamiebrynes7) [reply README.md]: Meh. I copied this from spatialos/UnrealGDK repo and we do link to it at the bottom, so don't mind.

Do you feel strongly about it?

**Pull request merged**